docs/openshift/deploy.md

```diff
@@ -0,0 +1,102 @@
+# Deploy an OpenShift Cluster
+
+## Install Pre-requisites
+
+All the commands in this guide require both the Azure CLI and `acs-engine`. Follow the [installation instructions to download acs-engine before continuing](../acsengine.md#install-acs-engine) or [compile from source](../acsengine.md#build-from-source).
+
+For installation instructions see [the Azure CLI GitHub repository](https://github.com/Azure/azure-cli#installation) for the latest release.
+
+## Overview
+
+`acs-engine` reads a cluster definition which describes the size, shape, and configuration of your cluster. This guide takes the default configuration of one master and two Linux agents. If you would like to change the configuration, edit `examples/openshift.json` before continuing.
+
+The `acs-engine deploy` command automates creation of a Service Principal\*, Resource Group and SSH key for your cluster. If operators need more control or are interested in the individual steps see the ["Long Way" section below](#acs-engine-the-long-way).
+
+**Note**: \* Doesn't work yet, Service Principal must be created beforehand and credentials must be provided in `examples/openshift.json` file.
+
+**Note:** ACS Engine creates a _cluster_; it _doesn't_ create an Azure Container Service resource.  So clusters that you create using the `acs-engine` command (or ARM templates generated by the `acs-engine` command) won't show up as ACS resources, for example when you run `az acs list`.  Think of `acs-engine` as the, er, engine which ACS uses to create clusters: you can use the same engine yourself, but ACS won't know about the results.
```

Comment:
I'm not quite sure what the point you're making is.  The tone in this paragraph needs to be straightened out.